### DISCUSSION TITLE
Changing default ports not changing

### DISCUSSION THREAD

**Discussion by random1planet** [louislam/dockge#546] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been reported before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### Description

I am trying to add to instances of radarr, so I changed the port numbers so they didnt conflict. but it doesnt seem to do what it should and they are not accessible on the advertised port. I also had the same issue trying to install jellyfn and emby, which by default use the same ports. 

### 👟 Reproduction steps

normal port config 7878:7878, changed to 7879:7879 in the compose file.

### 👀 Expected behavior

this should change the ui port to the new port

### 😓 Actual Behavior

cant access the url using the new port

### Dockge Version

1.4.2

### 💻 Operating System and Arch

bookworm

### 🌐 Browser

edge

**Comment by cyril59310** [comment, marked as answer]
Only the port before the colon needs to be changed (7879:7878). The port after the colon concerns the container and should not be changed.

7879 : 7878
host container

**Reply by random1planet** [reply]
Thank you so much for the explanation, I know you didn't have to and I really appreciate the help.
